platform/osx/detect.py

```diff
@@ -199,4 +202,19 @@ def configure(env):
         env.Append(CPPDEFINES=["VULKAN_ENABLED"])
         env.Append(LINKFLAGS=["-framework", "Metal", "-framework", "QuartzCore", "-framework", "IOSurface"])
         if not env["use_volk"]:
-            env.Append(LINKFLAGS=["-L$VULKAN_SDK_PATH/MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/", "-lMoltenVK"])
+            env.Append(LINKFLAGS=["-lMoltenVK"])
+            mvk_found = False
+            if env["VULKAN_SDK_PATH"] != "":
+                mvk_path = os.path.join(
+                    os.path.expanduser(env["VULKAN_SDK_PATH"]), "MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/"
```

Comment:
I'm not sure it's really worth changing further as far most users should have the SDK installed in the standard path and thus not need the manual option.

But semantically I would expect an upper case `VULKAN_SDK_PATH` option to be read from the OS environment, and an option that needs to be specified manually to SCons would be a lower case `vulkan_sdk_path=...`.

Reply:
It might be useful when using custom MKV build for debugging.

I guess `MACOS_SDK_PATH` also should be changed to a lower case, since it's an argument as well?

Reply:
> It might be useful when using custom MKV build for debugging.

Yeah I didn't mean removing the option, just that my suggestion to change its casing is maybe overkill :)
I do need the option for official builds since I have MVK in a non-standard path on Linux.

> I guess `MACOS_SDK_PATH` also should be changed to a lower case, since it's an argument as well?

Yeah I guess either that, or both could be read from the OS environment instead of SCons arguments.

Reply:
Maybe it was at some point, but I do not see anything that is getting it from environment variable (unlike `ANDROID_SDK_ROOT` which is, but argument probably can be lower case as well).

Reply:
Changed it to `vulkan_sdk_path`. I have not changed `MACOS_SDK_PATH` since there are similar upper case arguments used by iOS as well. Probably, all OS SDK paths should be changed to work in the same way on iOS/macOS/Android, but it's unrelated to this change.